integtest/spec/helper/sh.rb

```diff
@@ -15,18 +15,6 @@ def sh(cmd)
     out
   end
 
-  ##
-  # Init a git repo in root and commit any files in it.
-  def init_repo(root)
```

Comment:
I've moved this to the `Source` object because we'll want to initialize many repos when testing `--all`